airflow/www_rbac/utils.py

```diff
@@ -489,4 +491,40 @@ def is_utcdatetime(self, col_name):
                 isinstance(obj.impl, UtcDateTime)
         return False
 
+    # This is a local fix until https://github.com/dpgaspar/Flask-AppBuilder/pull/1493 is merged and released.
```

Comment:
Hmm, that comment they mention, that fix is merged and released

Reply:
Yea if you update to the latest FAB version, you won't need that

Reply:
PR: https://github.com/apache/airflow/pull/11884

Reply:
Cool, thanks - will follow up with a revert + version bump next week